_posts/2023-12-13-gift-guide.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Compiler's 2023 Holiday Gift Guide"
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler's got you covered! Here is this year's round-up of our favorite transit-related gift ideas.
+*(P.S. If you didn't make the holiday cut off this year, we think these make great gifts year-round!)*"
+description: ""
+author:
+excerpt: "Learn how to write a blog post, include images, blockquotes and more."
```

Comment:
This is what shows up here 
<img width="831" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/3513385f-8891-4499-ae65-e3a044f6e564">